### DISCUSSION TITLE
Natural Boundary Condition for Potential?

### DISCUSSION THREAD

**Discussion by ghost** [shannon-lab/zapdos#120] [Q&A Getting Started]
What is the natural boundary condition for potential? My best guess is that the dV/dx = 0 at the boundary.

**Comment by csdechant** [comment]
Just to double check, by natural boundary condition, you mean at a boundary where the potential is not restricted (e.g. a boundary that represents open air). If so, then yes, the BC would be grad(V) = 0.

**Reply by ghost** [reply]
At a given boundary, grad(V) = 0 holds even if the plasma species have specified BCs on that boundary unless a BC for the potential specifically is provided?

**Reply by csdechant** [reply]
Could you please give an example of when one might want to define a BC for the plasma species, but not the potential? (Are you still using one of Zapdos's flux BC for the plasma species, or are you defining the species with a new BC)
